### ISSUE TITLE
Find a better way to present reusable blocks in the inserter

### ISSUE THREAD

**Issue by ZebulanStanphill** [WordPress/gutenberg#22860]
Currently, reusable blocks are really buried in the inserter. They're located at the bottom of the "Blocks" tab, and with no way to filter or collapse categories, you have to scroll really far to even see them.

I think reusable blocks ought to be split out into their own primary category in the inserter, similar to block patterns. It's worth noting that template parts will likely get their own primary category soon.

The problem, of course, is that there's almost no room for any more tabs, especially when you take into account some languages using much longer words for certain phrases.

Because of this, I think we need to move away from tabs to something with more room for expansion. My current thinking is that a `<select>` dropdown would work well; it would make it easy to fit in 4 or more general categories, e.g.:

- Blocks
- Patterns
- Reusable Blocks
- Template Parts

I consider this a high priority issue, since due to the lack of collapsible block categories in the current inserter, it is way harder to find reusable blocks in `master` compared to current core WP.

**Comment by paaljoachim** [comment]
Here are mockups to the drop down idea.

![Block-Panel-Drop-Down-Closed](https://user-images.githubusercontent.com/5323259/83671841-9e428c80-a5d5-11ea-86ce-d9bdbff33776.jpg)

![Block-Panel-Drop-Down-Open](https://user-images.githubusercontent.com/5323259/83671863-a5699a80-a5d5-11ea-9b56-4865529bbbd1.jpg)

**Comment by ZebulanStanphill** [comment]
I think there should probably be a visible text label for the dropdown for better accessibility and clarity, but other than that, that's pretty much exactly what I was thinking. :+1:
